setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup
 import os
 
-PACKAGE_DIRS = ['local_write_example', 'fx_rates_example', 'mta', 'ip_to_country']
+PACKAGE_DIRS = ['local_write_example', 'fx_rates_example', 'mta', 'ip_to_country', 'wikipedia_word_frequency']
```

Comment:
Does this need to be added here?

Reply:
@tbantle22 sorry for the delayed response, yes it does, at least if you don't want local import errors. Airflow uses local import, so not needed there. I should actually fix this, as I've botched this part of the setup.